apps/checker/test/scala/services/EntityHelperTest.scala

```diff
@@ -1,55 +0,0 @@
-package services
```

Comment:
IIUC, if we keep the code in `EntityHelper` above, perhaps best to keep the test! Or, we could remove them both?